### ISSUE TITLE
Accessibility audit - address keyboard navigation issues

### ISSUE THREAD

**Issue by kewitham** [usdoj-crt/beta-ada#588]
Per Terri's Accessibility Review of Netlify:
1. ![image](https://user-images.githubusercontent.com/18104884/230459640-bb803455-9129-450f-afb0-5f76750e0991.png)
Not keyboard accessible
2. ![image](https://user-images.githubusercontent.com/18104884/230459723-47eb0382-3085-4bb1-a3a7-4a1fc8e5a53b.png)
No visual focus on tab
3. ![image](https://user-images.githubusercontent.com/18104884/230459803-edefa1a4-f534-4858-a41c-417559846fdc.png)
Once a keyboard user tabs into the editor (in rich text mode) they cannot get back out without arrowing all the way to the bottom of the text in the window. Example:  Escape could get the focus off the input and allow the user to tab to the next user interface component.

**Comment by kewitham** [comment]
[PR](https://github.com/decaporg/decap-cms/pull/6757) in review with Decap team

**Comment by kewitham** [comment]
PR merged
